### ISSUE TITLE
Use of option att for pre-trained networks

### ISSUE THREAD

**Issue by JoMe2704** [ZhaoJ9014/face.evoLVe.PyTorch#142]
In your code (for instance extract_feature), you use a parameter tta. If it is set, the embeddings of both the imag and its flipped version are computed and added. 
`if tta:
            emb_batch = backbone(ccropped.to(device)).cpu() + backbone(flipped.to(device)).cpu()`

Have the pre-trained models beed trained on this variant? Should I expect an improved accuracy when using this option?

**Comment by maywander** [comment]
So have you solved the problem？can you share the difference between setting tta=true and tta=false

**Comment by JoMe2704** [comment]
Yes and No. I have evaluated the accuracy with and without this option, and didn't obeserve any improvement (on the contrary: a slight decrease). Thus, I assume that the pretrained models have been trained with this option turned off.  

Nevertheless, if you plan to train your own model, it might be worth checking out.
